examples/minimum-viable-router/src/main.rs

```diff
@@ -0,0 +1,85 @@
+use route_rs_packets::EthernetFrame;
+use route_rs_runtime::link::primitive::{InputChannelLink, OutputChannelLink};
+use route_rs_runtime::link::{LinkBuilder, PacketStream, TokioRunnable};
+use tokio::task::JoinHandle;
+
+mod route;
+
+fn main() {
+    let mut runtime = tokio::runtime::Builder::new()
+        .threaded_scheduler()
+        .enable_all()
+        .build()
+        .unwrap();
+
+    let ingress_wan = crossbeam::channel::unbounded();
+    let ingress_lan = crossbeam::channel::unbounded();
+    let ingress_host = crossbeam::channel::unbounded();
+
+    let egress_wan = crossbeam::channel::unbounded();
+    let egress_lan = crossbeam::channel::unbounded();
+    let egress_host = crossbeam::channel::unbounded();
+
+    drop(ingress_wan.0);
```

Comment:
Why are we doing this? We can just let Rust do the dropping for us? Also won't the recv close as soon as we do this?

Reply:
That is the point. It looks a little silly here, but if we actually had packets to pass through, this would allow us to run a test in the `main()`. If we don't explicitly drop the channels, `run_route()` will never exit because nothing stops the router.